pkgs/applications/networking/cluster/kubectl-ktop/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "ktop";
+  version = "0.3.0";
+
+  src = fetchFromGitHub {
+    owner = "vladimirvivien";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-9TKXOKGsxtmyTGGVq7GDy4v3I9BF5SDc48dhrj1pw9Q";
+  };
+
+  vendorSha256 = "sha256-IbWdq6t9tgly7MRRIRJK5sDHus+sF0AT843aI4quO3c=";
+
+  postConfigure = ''
+    rm -rf ./.ci ./hack
+  '';
+
+  doCheck = false;
```

Comment:
Even if the upstream project has no tests at the moment, you can still just leave this on.
It won't error out just because of this.

And because `true` is the default for this flag here, you can just remove it all together:
```suggestion
```